src/tilemaps/components/HexagonalGetTileCorners.js

```diff
@@ -35,13 +35,13 @@ var HexagonalGetTileCorners = function (tileX, tileY, camera, layer)
     }
 
     //  Sets the center of the tile into tempVec
+    var tempVec = new Phaser.Math.Vector2;
     var center = HexagonalTileToWorldXY(tileX, tileY, tempVec, camera, layer);
 
     var corners = [];
 
-    //  Hard-coded orientation values for Pointy-Top Hexagons only
```

Comment:
Why remove a useful comment?

Reply:
The comment is contradictory, the value is not only used for Pointy-Top Hexagons, but an aid to calculating the long diagonals. Better comment: We assume a pointy-top orientation.